logging/logging_test.go

```diff
@@ -242,6 +325,87 @@ func TestLogAndEntries(t *testing.T) {
 	}
 }
 
+func TestLogWithMinimalSeverity(t *testing.T) {
```

Comment:
nit: it looks like a great opportunity for refactoring since both `TestLogSyncWithMinimalSeverity()` and `TestLogWithMinimalSeverity()` have very similar code lines.

Reply:
Ok. I will try to do something with it.

Reply:
Done